src/graph-helpers/graph-model.js

```diff
@@ -0,0 +1,15 @@
+export default class GraphModel {
+  constructor(attrs) {
+    this.attrs = attrs;
+
+    Object.keys(this.attrs).filter(key => {
+      return !(key in this);
```

Comment:
The whole point is that you can override a getter with your own to fill in business logic. The point is for someone to be able to `class MyThing extends GraphModel`, and it builds out the proxy getters, and allows you to write your own special ones that rely on `attrs`